authbridge/authlib/listener/reverseproxy/transparent_inbound_mtls_test.go

```
@@ -0,0 +1,543 @@
+	src := newTestSVIDSource(t, "spiffe://example.org/ns/team1/sa/agent")
+	metrics := &authtls.Metrics{}
+	srv, err := NewTransparentServer(inboundPipelineFromAuth(t, a), nil, &MTLSOptions{
+		Source:  src,
+		Strict:  strict,
+		Metrics: metrics,
+	})
+	if err != nil {
+		t.Fatalf("NewTransparentServer: %v", err)
+	}
+
+	tcpLn, err := net.Listen("tcp", "127.0.0.1:0")
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)